### PR TITLE
chore: more proposal scripts to update staking limits

### PR DESCRIPTION
**Pull request by kupermind**
- More proposal scripts to update staking limits.

### CODE REVIEW

**Review by 77ph — DISMISSED** [review]

**Review by 77ph — APPROVED** [review]

File: scripts/deployment/l2/globals_arbitrum_one.json [hunk 1]
```diff
@@ -1 +1 @@
...
...L2","serviceRegistrySymbol":"AUTONOLAS-SERVICE-L2-V1","timelockAddress":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","bridgeMediatorAddress":"0x4d30F68F5AA342d296d4deE4bB1Cacca912dA70F","serviceRegistryAddress":"0xE3607b00E75f6405248323A9417ff6b39B244b50","gnosisSafeMultisigImplementationAddress":"0x63e66d7ad413C01A7b49C7FF4e3Bb765C4E4bd1b","gnosisSafeSameAddressMultisigImplementationAddress":"0xBb7e1D6Cb6F243D6bdE81CE92a9f2aFF7Fbe7eac","operatorWhitelistAddress":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","serviceRegistryTokenUtilityAddress":"0x3d77596beb0f130a4415df3D2D8232B3d3D31e44","serviceManagerTokenAddress":"0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5","olasAddress":"0x064F8B858C2A603e1b106a2039f5446D32dc81c1","multisigProxyHash130":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000","stakingTokenAddress":"0x04b0007b2aFb398015B76e5f22993a1fddF83644","stakingNativeTokenAddress":"","minStakingDepositLimit":"10000000000000000000000","timeForEmissionsLimit":"2592000","numServicesLimit":"100","apyLimit":"3000000000000000000","stakingVerifierAddress":"0x7Fd1F4b764fA41d19fe3f63C85d12bf64d2bbf68","stakingFactoryAddress":"0xEB5638eefE289691EcE01943f768EDBF96258a80"}
```
Comment (DavidMinarsch) [inline scripts/deployment/l2/globals_arbitrum_one.json:1]: timelockAddress why added?

File: scripts/deployment/l2/globals_base_mainnet.json [hunk 2]
```diff
@@ -1 +1 @@
...
...ySymbol":"AUTONOLAS-SERVICE-L2-V1","bridgeMediatorAddress":"0xE49CB081e8d96920C38aA7AB90cb0294ab4Bc8EA","olasAddress":"0x54330d28ca3357F294334BDC454a032e7f353416","multisigProxyHash130":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000","serviceRegistryAddress":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","operatorWhitelistAddress":"0x3d77596beb0f130a4415df3D2D8232B3d3D31e44","serviceRegistryTokenUtilityAddress":"0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5","serviceManagerTokenAddress":"0x63e66d7ad413C01A7b49C7FF4e3Bb765C4E4bd1b","gnosisSafeMultisigImplementationAddress":"0xBb7e1D6Cb6F243D6bdE81CE92a9f2aFF7Fbe7eac","gnosisSafeSameAddressMultisigImplementationAddress":"0xFbBEc0C8b13B38a9aC0499694A69a10204c5E2aB","stakingTokenAddress":"0xEB5638eefE289691EcE01943f768EDBF96258a80","stakingNativeTokenAddress":"","minStakingDepositLimit":"10000000000000000000000","timeForEmissionsLimit":"2592000","numServicesLimit":"100","apyLimit":"3000000000000000000","stakingVerifierAddress":"0x10c5525F77F13b28f42c5626240c001c2D57CAd4","stakingFactoryAddress":"0x1cEe30D08943EB58EFF84DD1AB44a6ee6FEff63a","L1CrossDomainMessengerProxyAddress":"0xE49CB081e8d96920C38aA7AB90cb0294ab4Bc8EA"}
```
Comment (DavidMinarsch) [inline scripts/deployment/l2/globals_base_mainnet.json:1]: L1CrossDomainMessengerProxyAddress why addded?

File: scripts/deployment/l2/globals_celo_mainnet.json [hunk 3]
```diff
@@ -1 +1 @@
...
...02ED2cA2d42104F621f448A2cE1bC8Fb7","olasAddress":"0xaCFfAe8e57Ec6E394Eb1b41939A8CF7892DbDc51","multisigProxyHash130":"0xb89c1b3bdf2cf8827818646bce9a8f6e372885f8c55e5c07acbd307cb133b000","serviceRegistryAddress":"0xE3607b00E75f6405248323A9417ff6b39B244b50","operatorWhitelistAddress":"0x3C1fF68f5aa342D296d4DEe4Bb1cACCA912D95fE","serviceRegistryTokenUtilityAddress":"0x3d77596beb0f130a4415df3D2D8232B3d3D31e44","serviceManagerTokenAddress":"0x34C895f302D0b5cf52ec0Edd3945321EB0f83dd5","gnosisSafeMultisigImplementationAddress":"0x63e66d7ad413C01A7b49C7FF4e3Bb765C4E4bd1b","gnosisSafeSameAddressMultisigImplementationAddress":"0xBb7e1D6Cb6F243D6bdE81CE92a9f2aFF7Fbe7eac","stakingTokenAddress":"0xe1E1B286EbE95b39F785d8069f2248ae9C41b7a9","stakingNativeTokenAddress":"","minStakingDepositLimit":"10000000000000000000000","timeForEmissionsLimit":"2592000","numServicesLimit":"100","apyLimit":"3000000000000000000","stakingVerifierAddress":"0xc40C79C275F3fA1F3f4c723755C81ED2D53A8D81","stakingFactoryAddress":"0x1c2cD884127b080F940b7546c1e9aaf525b1FA55","wormholeL1MessageRelayerAddress":"0x27428DD2d3DD32A4D7f7C497eAaa23130d894911","wormholeMessengerAddress":"0x397125902ED2cA2d42104F621f448A2cE1bC8Fb7"}
```
Comment (DavidMinarsch) [inline scripts/deployment/l2/globals_celo_mainnet.json:1]: wormholeL1MessageRelayerAddress":"0x27428DD2d3DD32A4D7f7C497eAaa23130d894911","wormholeMessengerAddress":"0x397125902ED2cA2d42104F621f448A2cE1bC8Fb7" why added?

File: scripts/deployment/l2/globals_arbitrum_one.json (hunk 1 above)
Comment (kupermind) [reply scripts/deployment/l2/globals_arbitrum_one.json:1]: This is needed for L1-L2 interactions and proposals via scripts

File: scripts/deployment/l2/globals_base_mainnet.json (hunk 2 above)
Comment (kupermind) [reply scripts/deployment/l2/globals_base_mainnet.json:1]: This is needed for L1-L2 interactions and proposals

File: scripts/deployment/l2/globals_celo_mainnet.json (hunk 3 above)
Comment (kupermind) [reply scripts/deployment/l2/globals_celo_mainnet.json:1]: This is needed for L1-L2 interactions and proposals

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**